sdk/appconfiguration/Azure.Data.AppConfiguration/Azure.Data.AppConfiguration.sln

```diff
@@ -15,6 +15,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Identity", "..\..\ide
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Core.TestFramework", "..\..\core\Azure.Core.TestFramework\src\Azure.Core.TestFramework.csproj", "{1FC8A3EA-3C0D-4DDF-B710-A7091F2CEBB1}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Azure.Data.AppConfiguration.Perf", "perf\Azure.Data.AppConfiguration.Perf.csproj", "{034EF1C7-2953-49FD-BC45-9C3FF5986883}"
```

Comment:
Not sure how this guy got the Guid of the classic csproj project type. Might be a copy-paste issue. Should be `9A19103F` like the others.